Documentation/reports.md

```diff
@@ -242,9 +242,8 @@ spec:
 ```
 
 ### runImmediately
-
-Set `runImmediately` to `true` to run the report immediately with all available data, regardless of the `reportingStart` or `reportingEnd` values, and without checking if there is any data for the report period.
-For reports with a schedule set, it will not wait for each period's reportingEnd to elapse before processing and all reportPeriods between `reportingStart` and `reportingEnd`.
+Set `runImmediately` to `true` to run the report immediately, this is a simple way to have a report run right away without setting schedule values that would cause it to do so.
```

Comment:
When `runImmediately` is set to `true`, the report will be run immediately. This is a simple way to have a report get processed and queued right away without setting schedule values that could cause it to get requeued.

**Note**: You must set a `reportingEnd` value when `runImmediately` is set to `true`. If you do not specify a `reportingStart` value, the current time will be used.

Set `runImmediately` to `true` to run the report immediately with all available data, regardless of the `reportingStart` or `reportingEnd` values, and without checking if there is any data for the report period.

For reports with a schedule specified, it will not wait for each period's `reportingEnd` to elapse before processing and all the reporting periods between `reportingStart` and `reportingEnd`.

Reply:
I was thinking something along the lines of that, but I'm not sure how to incorporate a description that handles both a run-once and a scheduled report so it's clear enough to the user what the functionality of `runImmediately` does.